Fakebook.Posts/Fakebook.Posts.RestApi/Controllers/PostsController.cs

```diff
@@ -358,22 +358,23 @@ public async Task<ActionResult> UploadPicture(IFormFile file, string userId)
         /// <br/>200 OK on success
         /// </returns>
         [Authorize]
-        [HttpGet("newsfeed")]
+        [HttpPost("newsfeed")]
         [ProducesResponseType(StatusCodes.Status200OK)]
-        public async Task<IActionResult> GetNewsfeedAsync()
+        public async Task<IActionResult> GetNewsfeedAsync(NewsFeedDto newsfeedemails)
```

Comment:
it's not the client's job to decide what emails to build the news feed from, it's the business logic on this service's job.

Reply:
While I agree this isn't ideal., there isn't much that can be done at this point. Ideally, we should create a new microservice to handle followers and said business logic, but unfortunately we don't have the luxury of time to do so. At our current point, we would either have to revert back to coupling followers to posts (a bad idea) or having the client send the users to show in their news feed as we are doing in this PR (a less bad idea in my opinion). Having the client send the users also allows for more flexibility not directly tied, such as for Facebook-like Groups and advanced user-feed searching (eg. posts from two users you aren't following).

Reply:
that flexibility all probably belongs within the backend.
but okay, we can do it this way